src/main/java/com/flipkart/ranger/finder/AbstractZookeeperServiceRegistry.java

```diff
@@ -53,7 +55,7 @@ public void run() {
                 try {
                     updater.checkForUpdate();
                 } catch (Exception e) {
-                    e.printStackTrace();
+                    logger.error("Error while checking for update", e);
```

Comment:
See if printStackTrace is anywhere else .. let's remove it.

Reply:
Done, were present only in test cases